proto/stream_plan.proto

```diff
@@ -300,3 +300,42 @@ message StreamActor {
   // Placement rule for actor, need to stay on the same node as upstream.
   bool same_worker_node_as_upstream = 7;
 }
+
+enum FragmentType {
+  SOURCE = 0;
+  SINK = 1;
+  OTHERS = 2;
+}
+
+message StreamFragmentGraph {
+  message StreamFragment {
+    uint32 fragment_id = 1;
```

Comment:
Better to doc the behavior of fragment_id like: 0-based on frontend, and will be rewritten to global id on meta.